### DISCUSSION TITLE
How to share a glossary among multiple files

### DISCUSSION THREAD

**Discussion by JakeMomo** [typst-community/glossarium#62] [Q&A]
I'm writing a document with each chapter having a dedicated file. I tried sharing the glossary by writing the #print-glossary in a dedicated file and including it everywhere, but it doesn't work : the terms are not found by typst. Is there a way to do this ?

**Comment by slashformotion** [comment]
I have never tried this but if you don't run the show rule  at the start of each file, it will certainly not work.
